ads/_ping_.js

```diff
@@ -22,7 +22,11 @@ import {dev} from '../src/log';
  */
 export function _ping_(global, data) {
   global.document.getElementById('c').textContent = data.ping;
-
+  if (!data.nativeIntersectionObserver) {
+    const nullIO = () => {};
+    nullIO.prototype = null;
```

Comment:
hmmm, but it did work though. What should I do to assign the `window.IntersectionObserver` to null in this case. type-check require it to be a function, and this is my hack to fake native IntersectionObserver 😢